nrf_wifi/fw_if/umac_if/inc/fmac_api_common.h

```diff
@@ -241,6 +241,53 @@ enum nrf_wifi_status nrf_wifi_fmac_get_reg(struct nrf_wifi_fmac_dev_ctx *fmac_de
 enum nrf_wifi_status nrf_wifi_fmac_get_power_save_info(void *fmac_dev_ctx,
 						       unsigned char if_idx);
 
+#ifdef CONFIG_NRF700X_RAWDATA_TX
+/**
+ * @brief Get or set the current mode of operation
```

Comment:
How do we use this function to get the current mode?

Reply:
updated as discussed